theme/pxt.less

```diff
@@ -249,6 +249,14 @@ div.simframe > iframe {
     fill: none !important;
 }
 
+.blocklyFlyoutLabelText {
+    fill: white !important;
```

Comment:
theming can override this. The default fill is black for blockly, and it's more likely white would have better contrast here, hence why it's in pxt.less.